scripts/cmux-events-stress.py

```
@@ -0,0 +1,653 @@
+    def parse_reload_app_path(self, output: str) -> pathlib.Path | None:
+        lines = output.splitlines()
+        for index, line in enumerate(lines):
+            if line.strip() == "App path:" and index + 1 < len(lines):
+                raw_candidate = lines[index + 1].strip()
+                if raw_candidate:
+                    return pathlib.Path(raw_candidate).expanduser()
+        return None
+
+    def check_paths(self) -> None:
+        if self.socket_path.name in {"cmux.sock", "cmux-debug.sock"}:
+            raise StressFailure(f"refusing to use non-tagged socket path: {self.socket_path}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)